recipes/google-cloud-workflows/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "google-cloud-workflows" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/google-cloud-workflows-{{ version }}.tar.gz
+  sha256: 5a36556ccc55f716e98f3b5d7ea740bcd80ff8af10be704dea29f884b81cc918
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - google-api-core >=1.22.2,<2.0.0dev
+    - libcst >=0.2.5
+    - proto-plus >=1.4.0
+    - python >=3.6
+
+test:
+  imports:
+    - docs
+    - google
```

Is there a more specific import you can use? Perhaps:
```suggestion
    - google.cloud.tasks
```

---

Yep, that's a fair point.  I'll look into this but your suggestion seems good.